docs/topologic.rst

```diff
@@ -14,6 +14,7 @@ Subpackages
    topologic.embedding
    topologic.io
    topologic.iterators
+   topologic.partitioning
```

Comment:
Consider changing this to either partition or partitions but I don't feel particularly strong about this change.